### DISCUSSION TITLE
Verbindung wird instabiler

### DISCUSSION THREAD

**Discussion by Ascendor** [tbnobody/OpenDTU#410] [Q&A] [answered]
Hallo zusammen,

nachdem ich etwas Anlaufschwierigkeiten hatte (siehe https://github.com/tbnobody/OpenDTU/issues/332) lief mein OpenDTU-Modul sehr gut. Doch nun sehe ich eine immer schlechter werdende Verbindung. Nachdem ich anfangs stabil alle 5 Sekunden ein Update bekam, passiert jetzt oft minutenweise gar nichts. 
Das ist sehr "schön" im Home Assistant zu sehen:

26. November: 
![image](https://user-images.githubusercontent.com/2690622/205870441-bb3d4378-8032-4bc9-87e9-c377197364e9.png)

heute: 
![image](https://user-images.githubusercontent.com/2690622/205870517-1dbd6881-fbac-4412-aa2b-e9a38f76a193.png)

Hat soetwas schon mal jemand gesehen? Woran kann das liegen? (Der Standort der Geräte wurde natürlich nicht verändert. 
Ist das ein Beispiel, wo ein zusätzlicher Kondensator Abhilfe schaffen würde? Oder habt Ihr andere Ideen? 

Edit: Vielleicht noch ein wichtiger Hinweis: Die zwei Phasen vom heutigen Tag, wo mehr Messwerte sichtbar sind, sind jeweils nach einem Hard Reset (Netzstecker ziehen). Es scheint also nach einem Reboot ein paar Sekunden/Minuten gut zu sein, bevor es wieder einbricht.

**Comment by raccoon0815** [comment, marked as answer]
Hast du das Funkmodul mal getauscht. Ich hatte ein ähnliches Verhalten, bei mir war dann zwei Tage später das Modul defekt.

**Reply by Ascendor** [reply]
Funkmodul getauscht, Verkabelung neu gemacht, geht wieder. 
Nächstes Problem: https://github.com/tbnobody/OpenDTU/discussions/449 :)
